AppCenterDataStorage/AppCenterDataStorage/MSSerializableDocument.m

```diff
@@ -0,0 +1,15 @@
+#import "MSSerializableDocument.h"
+
+@implementation MSSerializableDocument
+
+- (nonnull instancetype)initFromDictionary:(nonnull NSDictionary *)__unused dictionary {
```

Comment:
We won't need `nonnull` if we keep `NS_ASSUME_NONNULL_BEGIN` in the `MSSerializableDocument.h` header file, right?